discovery-provider/src/queries/get_users_cnode.py

```diff
@@ -67,7 +72,10 @@ def get_users_cnode(cnode_endpoint_string, replica_type=ReplicaType.PRIMARY):
                     't.secondary1 = :cnode_endpoint_string OR '
                     't.secondary2 = :cnode_endpoint_string) AND'
             }
-            t.secondary1 is not NULL;
+            t.user_id > { prev_user_id } AND
+            t.secondary1 is not NULL
+            ORDER BY "user_id"
```

Comment:
this technically isn't necessary -- thoughts on keeping it?

Reply:
why isn't the order by necessary?

Reply:
ah wait I was thinking of something else. we totally need it for pagination